dds/InfoRepo/DCPSInfoRepo.mpc

```diff
@@ -1,4 +1,4 @@
-project(*Lib): dcpslib, iortable, dcps_inforepodiscovery, DCPSInfoRepo_bits, imr_client, svc_utils, install_lib {
+project(*Lib): dcpslib, dcps_tcp, iortable, dcps_inforepodiscovery, DCPSInfoRepo_bits, imr_client, svc_utils, install_lib {
```

Comment:
Is this change necessary?  I thought it was already covered by `DCPSInfoRepo_bits`.

Reply:
Hmm, I thought I reverted these...